R/display_html.R

```diff
@@ -0,0 +1,103 @@
+
+add_tag <- function(x, tag, args = ""){
+    
+    if(args != ""){
+        args <- paste0(" ", args)
+    }
+    
+    x <- sprintf( 
+        "<%s%s>%s</%s>", 
+        tag, 
+        args, 
+        paste0(x, collapse=""),
+        tag
+    )
+    return(x)
+}
+
+
+#' diffdf_css
+#' 
+#' Returns the CSS used by diffdf when rendering html output
+#' 
+#' @export
+diffdf_css <- function(){
```

Comment:
Added zebra though its a bit hard to know what to do with the header row, some options include:
![image](https://user-images.githubusercontent.com/8447209/105605856-4dfb8300-5d8e-11eb-9897-d875f76c6dda.png)

![image](https://user-images.githubusercontent.com/8447209/105605862-5784eb00-5d8e-11eb-8990-fe0aff5832f2.png)

![image](https://user-images.githubusercontent.com/8447209/105605873-67043400-5d8e-11eb-9b47-7baca1c2363a.png)


None are great but I have left it is as the first option for now with: 9fd40db

With regards to the alignment I am really struggling on how to address this as the implementation needs to handle a diverse array of tables with uncommon names i.e. just look at the 3 tables in the summary section:

![image](https://user-images.githubusercontent.com/8447209/105605981-9b77f000-5d8e-11eb-9bb7-d93b51e5ed91.png)

Likely this may need to be defined by the check function in its return object but then that also needs an update to the apis to pass that info along, likewise if we are aligning here then should we not be aligning in the ascii output as well?  My gut feeling is alignment needs to be spun out into its own dedicated ticket / PR

Reply:
I like the dark row for the header, I think it looks nice. Happy to have a seperate pr for alignment, I think it will definitely look better with correct alignment, but agree we need to handle different scenarios